code-of-conduct.md

```diff
@@ -33,29 +33,9 @@ This code of conduct applies both within project spaces and in public spaces
 when an individual is representing the project or its community.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting a project maintainer, Brandon Philips
-<brandon.philips@coreos.com>, and/or Rithu John <rithu.john@coreos.com>.
+reported by contacting a project maintainer listed at
+https://github.com/orgs/prometheus-operator/teams/kube-prometheus-reviewers/members. 
```

Comment:
This link isn't accessible to anyone outside of the prometheus-operator org

Reply:
Should it be https://github.com/orgs/prometheus-operator/people?
It only shows the ones that set visibility to public though

Reply:
My concern with that is that it will include maintainers of prometheus-operator that are not involved with kube-prometheus. Maybe we should keep a list of the maintainers available in the repository? Like for example the one in prometheus-operator: https://github.com/prometheus-operator/prometheus-operator/blob/master/MAINTAINERS.md

I had another look at the different team visibilities that we can set and teams can't be made available to folks outside of the org. https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility

Reply:
I changed that to link to https://github.com/prometheus-operator/prometheus-operator/blob/master/MAINTAINERS.md especially since we will be having both reviewer teams in there when https://github.com/prometheus-operator/prometheus-operator/pull/4157 lands.